oneflow/core/common/throw.h

```diff
@@ -13,34 +13,33 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-#ifndef ONEFLOW_API_PYTHON_FRAMEWORK_THROW_H_
-#define ONEFLOW_API_PYTHON_FRAMEWORK_THROW_H_
+#ifndef ONEFLOW_CORE_COMMON_THROW_H_
+#define ONEFLOW_CORE_COMMON_THROW_H_
 
 #include "oneflow/core/common/error.h"
 
 namespace oneflow {
 
-class Throw final {
- public:
-  Throw(const Error& error) : error_(error) {}
-  ~Throw() noexcept(false) { ThrowError(error_.error_proto()); }
```

Comment:
it is dangerous to throw in a dtor, the program will terminate if another exception is propagating at the same time, so I remove it